### ISSUE TITLE
[Question]: Link to Figma a11y design kit doesn't work

### ISSUE THREAD

**Issue by dtron** [IBMa/equal-access#964]
### Project

able site

### Description

(hopefully this is the right place to report this!)

On the [toolkit/design](https://www.ibm.com/able/toolkit/design/overview) page, the link to the **IBM Accessibility Design Kit for Figma** does not work. It appears to be a private file, or a link that isn't valid. 

To repro, try opening the link in a private browser window or make sure you're not signed in to Figma (the link may be valid for accounts inside the IBM org)

**Comment by marcjohlic** [comment]
@dtron Thank you for letting us know.  We're working to get the links updated on the website, but in the meantime here's direct link to the Accessibility Design Kit in the Figma community.  You should be able to duplicate the kit from this link:  https://www.figma.com/community/file/1118184491812988116

Please let us know if you have any issues accessing from there.

**Comment by dtron** [comment]
This community link does work for me -- thank you for the quick turnaround!
